.pre-commit-config.yaml

```diff
@@ -20,6 +20,13 @@ repos:
           - --expand-star-imports
           - --remove-duplicate-keys
           - --remove-unused-variables
+  - repo: https://github.com/Pierre-Sassoulas/copyright_notice_precommit
+    rev: 6f5a59c3e1cb0f20731eb1ff32622c9a2fc72d9a
+    hooks:
+      - id: copyright-notice
+        args: ["--notice=copyright.txt", "--enforce-all", "--autofix"]
+        exclude: tests/testdata
+        types: [python]
```

Comment:
Shouldn't this be all? Do we only add copyright to Python files?

Reply:
The issue is that if the comment (meaning ``#``) do not work the same it would be an issue. We only have copyright on the code right now.